service/samples/registry_bulk_identity_sample.js

```diff
@@ -0,0 +1,131 @@
+var outputContainerName = 'exportcontainer';
+var deviceFile = 'devices.txt';
+
+blobSvc.createContainerIfNotExists(inputContainerName, function (error) {
+    if(error) {
+        console.error('Could not create input container: ' + error.message);
+    } else {
+        var inputSasToken = blobSvc.generateSharedAccessSignature(inputContainerName, null, inputSharedAccessPolicy);
+        var inputSasUrl = blobSvc.getUrl(inputContainerName, null, inputSasToken);
+        blobSvc.createBlockBlobFromLocalFile(inputContainerName, deviceFile, deviceFile, function (error) {
+            if (error) {
+                console.error('Could not create devices.txt: ' + error.message);
+            } else {
+                blobSvc.createContainerIfNotExists(outputContainerName, function (error) {
+                   if (error) {
+                       console.error('Could not create output container: ' + error.message);
+                   } else {
+                       var outputSasToken = blobSvc.generateSharedAccessSignature(outputContainerName, null, outputSharedAccessPolicy);
+                       var outputSasUrl = blobSvc.getUrl(outputContainerName, null, outputSasToken);
+
+                    /**
+                     * There can only be one active job at a time, therefore, you can uncomment the export section and comment the import
+                     * session or vice versa depending on what part of the code you would like to test.
+                     */
+
+                    //    registry.exportDevicesToBlobByIdentity(outputSasUrl, false, function (error, result) {
+                    //        if (error) {
+                    //            console.error('Could not create export job: ' + error.message);
+                    //        } else {
+                    //            console.log('--------------\r\nDevices Export Job Identifier:--------------\r\n' + result);
+                    //            var jobId = result.jobId;
+                    //            var interval = setInterval(function () {
+                    //                registry.getJob(jobId, function (error, result) {
+                    //                   if (error) {
+                    //                       console.error('Could not get job status: ' + error.message + ' : ' + error.responseBody);
+                    //                   } else {
+                    //                       console.log('--------------\r\njob ' + jobId + ' status:\r\n--------------\r\n' + result);
+                    //                       var status = result.status;
+                    //                       if (status === "completed") {
+                    //                           clearInterval(interval);
+                    //                       }
+                    //                   }
+                    //               });
+                    //            }, 500);
+                    //        }
+                    //    });
```

Comment:
How about no commenting code and have this be an if/else controlled by a command parameter?

Reply:
Or create another clean sample. (registry_bulk_identity_export and _import)

---
In reply to: [388026474](https://github.com/Azure/azure-iot-sdk-node/pull/709#discussion_r388026474) [](ancestors = 388026474)

Reply:
I just copied the existing sample for this sample.  Let's have a conversation about this offline.

---
In reply to: [388035333](https://github.com/Azure/azure-iot-sdk-node/pull/709#discussion_r388035333) [](ancestors = 388035333,388026474)